Added json output for Status command

lulco opened the pull request:
@ricco24

lulco committed:
Added json output for Status command

ricco24 commented:
It looks good. But maybe better is to use some kind of reusable formatter object for different output formats. (json, table, text, etc.)

Are you preparing different output formats for other phoenix commands?

lulco committed:
- fixed tests for status command
- added tests for json output in status command
- removed exit from status command

lulco commented:
For now this is just prove of concept :) I am not sure it will be possible to create reusable formatter object.

lulco committed:
Merge branch 'master' into status_command_json_output

lulco committed:
Updated changelog

lulco committed:
Merge branch 'json_output_for_commands' into status_command_json_output

Conflicts:
	CHANGELOG.md
	src/Command/AbstractCommand.php

lulco committed:
Merge branch 'json_output_for_commands' into status_command_json_output

lulco committed:
Used method is default output
